README.md

```diff
@@ -1,88 +1,80 @@
+export S3_BUCKET=com.ft.coco-mongo-backup.prod;
+export S3_DOMAIN=s3-eu-west-1.amazonaws.com
+export S3_DIR=pre-prod-uk
+export AWS_ACCESS_KEY_ID=123
+export AWS_SECRET_ACCESS_KEY=456
+
+docker run --rm --env "MONGODB=$MONGODB" --env "S3_DOMAIN=$S3_DOMAIN" --env "S3_BUCKET=$S3_BUCKET" --env "S3_DIR=$S3_DIR" --env "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" --env "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" coco/mongo-hot-backup:2.0.0-productionize-rc7 backup --collections="upp-store/lists,upp-store/list-notifications"
 ```
 
-* <MONGODB_ADDRESSES> - The address to connect to MongoDB cluster
-* <S3_DOMAIN> - The domain name of S3 location where the restore should go
-* <S3_BUCKET> - The S3 bucket name
-* <ENVIRONMENT_TAG> - The S3 folder name, which should represent the environment tag
-* <AWS_ACCESS_KEY> - The AWS access key
-* <AWS_SECRET_KEY> - The AWS secret key
-* <app_version> - The Docker image version of the app. Latest if omitted
-* <db> - The DB under the collections are
-* <coll_nr> - The collection to be restored
-* <timestamp> - The timestamp of the backup date
-
-Example:
+You can also try `docker run --rm coco/mongo-hot-backup backup --help`
+
+### Restoring
+
 ```
-docker run \
-          -e MONGODB=$(for x in $(etcdctl ls /ft/config/mongodb);do echo -n $(etcdctl get $x/host):$(etcdctl get $x/port)"," ; done | sed s/.$//) \
-          -e S3_DOMAIN=s3-eu-west-1.amazonaws.com \
-          -e S3_BUCKET=com.ft.coco-mongo-backup.prod \
-          -e S3_DIR=/pre-prod-uk/ \
-          -e AWS_ACCESS_KEY_ID=$(/usr/bin/etcdctl get /ft/_credentials/aws/aws_access_key_id) \
-          -e AWS_SECRET_ACCESS_KEY=$(/usr/bin/etcdctl get /ft/_credentials/aws/aws_secret_access_key) \
-          coco/mongodb-hot-backup:v0.2.0 /mongodb-hot-backup restore upp-store/content,upp-store/lists,upp-store/notifications 2017-02-14T08-25-36
+export MONGODB=ip-172-24-11-64.eu-west-1.compute.internal:27018,ip-172-24-186-252.eu-west-1.compute.internal:27020,ip-172-24-74-51.eu-west-1.compute.internal:27019
+export S3_BUCKET=com.ft.coco-mongo-backup.prod;
+export S3_DOMAIN=s3-eu-west-1.amazonaws.com
+export S3_DIR=pre-prod-uk
+export AWS_ACCESS_KEY_ID=123
+export AWS_SECRET_ACCESS_KEY=456
+
+docker run --rm --env "MONGODB=$MONGODB" --env "S3_DOMAIN=$S3_DOMAIN" --env "S3_BUCKET=$S3_BUCKET" --env "S3_DIR=$S3_DIR" --env "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" --env "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" coco/mongo-hot-backup:2.0.0-productionize-rc7 restore --collections="upp-store/lists,upp-store/list-notifications" --date="2017-09-04T12-40-36"
 ```
+
+You can also try `docker run --rm coco/mongo-hot-backup restore --help`
+
+## Links
+
+* [mongodb backup/restore documentation on FT Technology's Google Sites](https://sites.google.com/a/ft.com/technology/systems/dynamic-semantic-publishing/extra-publishing/mongo-db-run-book/mongo-db-backup-restore)
```

Comment:
It seems we shouldn't put documentation here but instead have it here `https://sites.google.com/a/ft.com/universal-publishing`.

Reply:
will be removed.